src/helpers/hierarchy.ts

```diff
@@ -0,0 +1,89 @@
+import { ChildrenByParentId, Jurisdiction } from '../store/ducks/jurisdictions';
```

Comment:
@cKellyDesign after refactoring the IRS reporting table component I am not sure if we need these anymore.  What are your thoughts on this?

Reply:
Agreed, this is now unused and the first iterations of these are contained in the IRS Jurisdiction Selection `Plan` page, which needs to be refactored, at which point we will migrate such functions into helpers.